package.json

```diff
@@ -42,6 +43,10 @@
     "eslint-check":
       "eslint --print-config .eslintrc.json | eslint-config-prettier-check",
     "reformat": "node scripts/reformat-with-prettier.js",
+    "synk:test":
+      "lerna exec --bail=false --ignore salesforcedx-vscode -- snyk test --severity-threshold=medium --show-vulnerable-paths=false",
```

Comment:
The only reason that salesforcedx-vscode is ignored is because it's just an extension pack. Synk works by scanning the node_modules (so it needs to run after `npm install`).